### ISSUE TITLE
install fails when kubernetes == 1.16

### ISSUE THREAD

**Issue by kartoch** [jfelten/gitea-helm-chart#28]
```
helm install --values gitea-values.yaml --name gitea ./gitea-helm-chart
Error: validation failed: unable to recognize "": no matches for kind "Deployment" in version "extensions/v1beta1"
```

**Comment by jfelten** [comment]
Thanks for fixing - I merged the PR - can you close if xied?

**Comment by sfxworks** [comment]
```
helm install --values gittea-values.yaml --name gitea --namespace gittea jfelten/gitea
Error: validation failed: unable to recognize "": no matches for kind "Deployment" in version "extensions/v1beta1"
```
Seems to still be occuring on 1.16

```
kubectl version
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:36:53Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:27:17Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by martin-alt** [comment]
Error: validation failed: unable to recognize "": no matches for kind "Deployment" in version "extensions/v1beta1"

Hi - I'm pretty new to this, so pls forgive my fumblings in advance. I'm trying a test install on Minikube and had to drop down to the previous version of k8s to get Tiller to install correctly as I think Helm got mixed up by k8s 1.16, but am getting this same error. I did a 'debug' run and it seemed to be working with a Gitea version 1.60 rather than 1.93, so I'm wondering if maybe this Deploy error is related to that since now Deploy is in app/v1. I don't presume to know what I'm talking about, but any thoughts? Thanks in advance!

Edit for more detail. The values definitely call for Gitea 1.9.3 everywhere but instead it keeps grabbing 1.6. I notice in the readme that 1.6.1. is the default, maybe it's defaulting for some reason?

(base) mbp-jr:home$ helm install jfelten/gitea --debug --name gitea --namespace gitea-ns
[debug] Created tunnel using local port: '59562'
[debug] SERVER: "127.0.0.1:59562"
[debug] Original chart version: ""
[debug] Fetched jfelten/gitea to /Users/home/.helm/cache/archive/gitea-1.6.0.tgz
[debug] CHART PATH: /Users/home/.helm/cache/archive/gitea-1.6.0.tgz
Error: validation failed: unable to recognize "": no matches for kind "Deployment" in version "extensions/v1beta1"

Edit2: Sorry to be a pest, but I ran it again from a local download of your chart, no other changes, and it worked perfectly. Out of my depth guessing why — but in any event thanks very much for creating!
